types/component/ChatList/ChannelImage.types.ts

```diff
@@ -9,11 +9,11 @@ export type ChannelImageProps = React.FC & {
 };
 
 export type ChannelImageMainProps = {
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: 'TOPIC' | 'ANON_TOPIC' | 'GROUP' | 'GROUP_INFO';
   image?: string;
   style?: StyleProp<ImageStyle>;
 };
 
 export type ChannelImageBadgeProps = {
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: 'TOPIC' | 'ANON_TOPIC' | 'GROUP' | 'GROUP_INFO';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new types 'TOPIC' and 'ANON_TOPIC' have been added to the `type` field of both `ChannelImageMainProps` and `ChannelImageBadgeProps`. This change seems to be in line with the PR's purpose of introducing anonymous topic chat functionality. However, it's important to ensure that all parts of the codebase that use these types are updated accordingly to handle these new types. If not, this could lead to unexpected behavior or crashes.

```diff
- type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+ type: 'TOPIC' | 'ANON_TOPIC' | 'GROUP' | 'GROUP_INFO';
```
Please make sure that all components using these types are updated to handle 'TOPIC' and 'ANON_TOPIC'.


<!-- This is an auto-generated comment by OSS CodeRabbit -->